src/main/java/za/ac/sun/cs/adversarial/domain/Move.java

```diff
@@ -0,0 +1,39 @@
+package za.ac.sun.cs.adversarial.domain;
+
+import java.util.Optional;
+
+/**
+ * The class representing moves made in the (m,n,k)-domain, including
+ * information such as the row and column of the move, as well as the
+ * identifier of the player who made the move.
+ */
+public class Move {
+    private final int row;
+    private final int column;
+    private final Optional<Integer> player;
```

Comment:
I'm not sure that we will even need to keep track of the player that made the move, but I have added this as an `Optional` so that we have the option.